src/modules/createtr.js

```diff
@@ -0,0 +1,38 @@
+const renderContainer = async (gameLInk) => {
+  const res = await fetch(gameLInk);
+  const resData = await res.json();
+  // .then((resData) => {
```

Comment:
- [ ] I think it's better to remove unused commented code from your app. Makes your repo looks more professional for recruiters and hiring managers.